packages/core/src/lib/toast/toast-item.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import type { ToastVariantType } from './variants.ts';
+import { ToastBanner } from '$lib/toast';
+
+export let variant: ToastVariantType;
+export let message: string;
+export let action: { text: string; handler: () => unknown } | undefined =
+  undefined;
+export let dismiss: () => unknown;
+</script>
+
+<ToastBanner
+  {variant}
+  cx="pointer-events-auto relative"
+  on:close={dismiss}
+  {dismiss}
+>
+  <svelte:fragment slot="message">
+    {message}
+  </svelte:fragment>
+  <svelte:fragment slot="action">
+    {#if action}
```

Comment:
This `if` looks too low, this usage of `slot="action"` is going to trigger `if $$slots.action` in `<ToastBanner>`. I think this mostly relates to me comment above about there being one two many components in play in this internal structure

Reply:
I can move everything in this file to <ToastBanner>